src/pages/run-a-node-grants/index.tsx

```diff
@@ -0,0 +1,332 @@
+                      custom build.
+                    </ListItem>
+                    <ListItem>
+                      Applicants are required to provide details about their understanding of nodes
+                      and clients.
+                    </ListItem>
+                    <ListItem>
+                      Applicants must detail their intended output for the Ethereum ecosystem by
+                      running a node and how they will share this impact with the community.
+                    </ListItem>
+                  </List>
+                </section>
+
+                <section id='helpful-resources' ref={ref6}>
+                  <PageSection mb={6}>Helpful Resources</PageSection>
+
+                  <List>
+                    <ListItem>
+                      <Link
+                        fontWeight={700}
+                        color='brand.orange.100'
+                        href='https://docs.dappnode.io/user/faq/general/'
+                        isExternal
+                      >
+                        Dappnode FAQ
+                      </Link>
+                    </ListItem>
+                    <ListItem>
+                      <Link
+                        fontWeight={700}
+                        color='brand.orange.100'
+                        href='https://ethereum.org/en/developers/docs/nodes-and-clients/'
+                        isExternal
+                      >
+                        Ethereum Nodes and Clients
+                      </Link>
+                    </ListItem>
+                    <ListItem>
+                      <Link
+                        fontWeight={700}
+                        color='brand.orange.100'
+                        href='https://nodewatch.io'
+                        isExternal
+                      >
+                        NodeWatch
+                      </Link>
+                    </ListItem>
+                    <ListItem>
+                      <Link
+                        fontWeight={700}
+                        color='brand.orange.100'
+                        href='https://ethernodes.org'
+                        isExternal
+                      >
+                        EtherNodes
+                      </Link>
+                    </ListItem>
+                    <ListItem>
+                      <Link
+                        fontWeight={700}
+                        color='brand.orange.100'
+                        href='https://panarama.xyz'
+                        isExternal
+                      >
+                        Panarama
+                      </Link>
+                    </ListItem>
+                  </List>
+                </section>
+
+                <section id='next-steps-and-support' ref={ref7}>
+                  <PageSection mb={6}>Next steps and support</PageSection>
+
+                  <PageText mb={6}>
+                    For general support questions about your submission, please email PGR INBOX or{' '}
```

Comment:
Whats `PGR INBOX`?

Reply:
lol agreed, it's odd. remove that text please. so just `For general support questions about your submission, please email grant-rounds@ethereum.org`